packages/jest-util/src/importModule.ts

```diff
@@ -0,0 +1,27 @@
+import {pathToFileURL} from 'url';
+import interopRequireDefault from './interopRequireDefault';
+
+export default async function importModule<T>(filePath: string): Promise<T> {
```

Comment:
Can you verify the path passed is absolute? Relative paths will resolve wrongly (all these paths _should_ be absolute, but throwing early if not is a good idea)